### DISCUSSION TITLE
I joined another server on my alt for it to snipe but it doesn't update in logs as it still says "Sniping in 2 servers" (not updated, im now in 3)

### DISCUSSION THREAD

**Discussion by CheeseTheWeese** [GiorgioBrux/nitro-sniper-enhanced#125] [Q&A]
is the log supposed to update when i join a new server or will it still snipe in 3 servers, it just doesnt update?

**Comment by GiorgioBrux** [comment]
If you restart the sniper it should update. If it doesn't don't worry, it will still work as intended.
